superset-frontend/src/explore/components/ControlPanelsContainer.jsx

```diff
@@ -182,6 +186,15 @@ class ControlPanelsContainer extends React.Component {
   }
 
   render() {
+    const controlPanelRegistry = getChartControlPanelRegistry();
+    if (
+      !controlPanelRegistry.has(this.props.form_data.viz_type) &&
+      this.context.loading
+    ) {
+      // TODO [dynamic-plugins] replace with a snazzy loading spinner
```

Comment:
You could just `import Loading from 'src/components/Loading';` and then here just `return <Loading />;`